solutions/src/main/java/ru/sberbank/school/task11/Streams.java

```diff
@@ -0,0 +1,179 @@
+package ru.sberbank.school.task11;
+
+import lombok.NonNull;
+
+import java.util.*;
+import java.util.function.Function;
+import java.util.function.Predicate;
+import java.util.stream.Stream;
+
+/**
+ * Облегченная версия класса {@link Stream}
+ */
+public class Streams<T> {
+    private final List<T> list = new ArrayList<>();
+
+    private final List<Function <List<T>, List<?>>> functions = new ArrayList<>();
```

Comment:
GenericWhitespace &#39;&lt;&#39; is preceded with whitespace.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a1180118198>*